vembrane/modules/filter.py

```diff
@@ -62,46 +62,89 @@ def add_subcommmand(subparsers):
         help="Keep all annotations of a variant if at least one of them passes "
         "the expression.",
     )
+    parser.add_argument(
+        "--preserve-order",
+        default=False,
+        action="store_true",
+        help="Make sure that the variant order of the output matches that of the \
+              input. For performance reasons only make use of this option, if the \
+              input might contain breakends (BNDs) as the order of all other variants \
+              are preserved anyway.",
```

Comment:
```suggestion
        help="Ensures that the order of the output matches that of the \
              input. This is only useful if the input contains breakends (BNDs) \
              since the order of all other variants is preserved anyway.
```